packages/medusa/src/strategies/product-export.ts

```diff
@@ -0,0 +1,670 @@
+import { Request, Response } from "express"
```

Comment:
suggestion(non-blocking): I think we should group the batch strategies in a folder inside strategies, i.e. `strategeis/batch` or something similar in order to prevent clutter in the folder.

We can discuss how we want to structure this, my approach initially was to separate them according to domain, becuase you could have batches that are not imports/exports explicitly. that would mean placing this in a `strategies/batch/product` folder. 

wdyt?

Reply:
Think this is an excellent idea!

Reply:
yes indeed nice idea :) I will do that